src/test/java/htsjdk/samtools/reference/SamLocusAndReferenceIteratorTest.java

```diff
@@ -0,0 +1,44 @@
+package htsjdk.samtools.reference;
+
+import htsjdk.HtsjdkTest;
+import htsjdk.samtools.SamReader;
+import htsjdk.samtools.SamReaderFactory;
+import htsjdk.samtools.util.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+/**
+ * Created by farjoun on 6/1/18.
```

Comment:
I'm not a fan of these types of comments, git should track the provenance just fine.  That's very much just a personal preference though.